supabase/migrations/20260410065000_create_conversation_ledger.sql

```
@@ -0,0 +1,80 @@
+
+ALTER TABLE public.conversation_events ENABLE ROW LEVEL SECURITY;
+
+CREATE POLICY "Patients can view their own conversation events"
+    ON public.conversation_events
+    FOR SELECT
+    USING (auth.uid() = patient_id);
+
+CREATE POLICY "Server can insert conversation events"
+    ON public.conversation_events
+    FOR INSERT
+    WITH CHECK (true); -- Real validation will happen via Server-side constraints/APIs
```

SQLFluff (4.1.0)

[error] Line 18, Position 5: Found unparsable section: "WHERE status = 'active';\n\nCREATE INDEX i..."

(PRS)